_offerings/offering-migrations.md

```diff
@@ -7,61 +7,63 @@ tag: Onboard
+In this session, GitHub’s experts will guide customers through the process of migrating their VCS data to GitHub Enterprise. The training covers best practices, tools, and techniques to ensure a smooth and efficient migration.
 
-## Target Audience
+### Topics
 
-- VCS Admins
-- GitHub Enterprise Admins
-- DevOps Engineers
+- Review of current VCS environment  
+- Migration plan development  
+- Enablement of migration tooling  
+- Proof of concept migration of several repositories  
+- Guidance and best practices for full repository set migration  
+- 30-days migration support with a GitHub migration specialist
 
-## Key Features and Benefits
+### Customer benefits
 
-- The Standard Migration scopes supported for this engagement are VCS sources listed below:
-  - GitHub (pre-exsiting versions of supported GitHub instances)
-  - GitLab
-  - BitBucket
-- Examine migration scenarios and best practices
-- Expert advice on approaches for source VCS to Git migration
-- Crafting of repeatable playbooks, enabling future self-serve migrations
+This offering will help customers:
 
-*_Note: Non listed VCS are treated as third-party sources and are not supported via our Standard Migration Engagement._
+- Improve understanding of the migration processes  
+- Enhance ability to manage and execute migrations  
+- Reduce risks and complexities associated with migrations  
+- Access expert guidance and support to accelerate the migration process
 
-## Schedule of Work
+### Learning objectives
 
-### Remote
+By the end of this training, learners will be able to:
 
-Additional scoping required
+- Evaluate their current VCS environment  
+- Develop a comprehensive migration plan with timelines  
+- Enable and operate migration tooling effectively  
+- Execute proof of concept migrations  
+- Apply best practices for full repository migrations
 
-### Onsite
+### Audience
 
-Additional scoping required
+**Required**:
 
-## Syllabus
+- DevOps Engineers  
+- GitHub Enterprise Admins  
+- Source VCS Admins
 
-In this engagement, a GitHub Expert Services Engineer will guide you through the process of migrating your VCS sources to the GitHub Enterprise target. This includes:
+**Optional**:
 
-- Documentation of migration process and considerations
-- Clean up of large binaries
-- Consolidation, clean up of branches
-- Version control history trimming
-- Review source VCS to Git migration utilities
-- Aid in source VCS to Git history migration dry-runs
-- Crafting of repeatable playbooks, enabling future self-serve migrations
+- Teams within the company who will perform the migration
 
-## Business Outcomes
+### Delivery details
 
-After this engagement, your team will be able to:
+- **Level:** Intermediate \[200\]  
+- **Offering type:** Training  
```

Comment:
@sparlant need your help to guide @lytcre on the wording for this offering please.